.ci/docker/ci_commit_pins/triton-xpu.txt

```diff
@@ -1 +1 @@
-b8c64f64c18d8cac598b3adb355c21e7439c21de
+dd072259e1df5c0cd42034d90ec5f464f22e5ae3
```

Comment:
Please pass all the torch-xpu-ops CI and dynamo benchmarks and then update the commit pin.